app/components/Exchange/BuySell.jsx

```diff
@@ -428,377 +385,284 @@ class BuySell extends React.Component {
-                                                ].get("symbol"),
-                                                arg: "asset"
-                                            },
-                                            {
-                                                type: "translate",
-                                                value: "exchange.deposit",
-                                                arg: "direction"
-                                            }
-                                        ]}
-                                    />
-                                </a>
-                            </div>
-                        ) : null}
-                        {this.props.onBorrow ? (
-                            <div className="float-right buy-sell-deposit">
-                                <a onClick={this.props.onBorrow}>
-                                    <TranslateWithLinks
-                                        string="exchange.buysell_formatter"
-                                        noLink
-                                        keys={[
-                                            {
-                                                type: "asset",
-                                                value: this.props[
-                                                    isBid ? "base" : "quote"
-                                                ].get("symbol"),
-                                                arg: "asset"
-                                            },
-                                            {
-                                                type: "translate",
-                                                value: "exchange.borrow",
-                                                arg: "direction"
-                                            }
-                                        ]}
-                                    />
-                                </a>
-                            </div>
-                        ) : null}
-                    </div>
+                    </div> : null}
 
                     <form
                         className={
                             (!this.props.isOpen ? "hide-container " : "") +
                             "order-form"
                         }
+                        style={{fontSize: "14px"}}
                         noValidate
                     >
-                        <div className="grid-block vertical no-overflow no-padding">
-                            <div className="grid-block no-padding buy-sell-row">
-                                <div className="grid-block small-3 no-margin no-overflow buy-sell-label">
-                                    <Translate content="exchange.price" />:
-                                </div>
-                                <div className="grid-block small-5 no-margin no-overflow buy-sell-input">
-                                    <ExchangeInput
```

Comment:
Moving away from ExchangeInput here is problematic since a lot of work has gone into that component in order to validate user input as well as fixing some copy paste issues. After removing this you can enter numbers and multiple decimals etc. Please check if you can continue to use ExchangeInput instead here. Feel free to modify it to use Antd components if necessary.

Reply:
Resolved by including ExchangeInput with Antd components.